extensions/default/src/getCustomizationModule.tsx

```diff
@@ -29,315 +30,267 @@ export default function getCustomizationModule({ servicesManager, extensionManag
   return [
     {
       name: 'helloPage',
-      merge: 'Append',
       value: {
-        id: 'customRoutes',
-        routes: [
-          {
-            path: '/custom',
-            children: () => <h1 style={{ color: 'white' }}>Hello Custom Route</h1>,
+        customRoutes: {
+          routes: {
+            $push: [
+              {
+                path: '/custom',
+                children: () => <h1 style={{ color: 'white' }}>Hello Custom Route</h1>,
+              },
+            ],
           },
-        ],
+        },
       },
     },
-
     // Example customization to list a set of datasources
     {
       name: 'datasources',
-      merge: 'Append',
       value: {
-        id: 'customRoutes',
-        routes: [
-          {
-            path: '/datasources',
-            children: DataSourceSelector,
+        customRoutes: {
+          routes: {
+            $push: [
+              {
+                path: '/datasources',
+                children: DataSourceSelector,
+              },
+            ],
           },
-        ],
+        },
       },
     },
     {
       name: 'multimonitor',
-      merge: 'Append',
       value: {
-        id: 'studyBrowser.studyMenuItems',
-        customizationType: 'ohif.menuContent',
-        value: [
-          {
-            id: 'applyHangingProtocol',
-            label: 'Apply Hanging Protocol',
-            iconName: 'ViewportViews',
-            items: [
-              {
-                id: 'applyDefaultProtocol',
-                label: 'Default',
-                commands: [
-                  'loadStudy',
-                  {
-                    commandName: 'setHangingProtocol',
-                    commandOptions: {
-                      protocolId: 'default',
+        'studyBrowser.studyMenuItems': {
```

Comment:
Could we import complex items like this one from a study browser import so that they are defined alongside other study browser content?  I don't like having a mish mash of different things in this file.  It does need to bring things together, but that could be done effectively by using imports and treating the customizations basically like an export set of stuff.